From 8cf2f1e6bffefd13177a66d0565e99c3d57e3fa1 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Thu, 12 Mar 2026 10:59:00 +0000
Subject: [PATCH 6/6] kernel_configs: Lower 4.19 minimum to 4.19.127

Many MediaTek devices use 4.19-r kernels shipped with 4.19.127.
Since r/android-4.19 does not exist on 16 QPR2, lower the minimum
LTS requirement for s/android-4.19 to support these devices.

Change-Id: I44e4127ad2ec25f2272c38215615dc936cb40b4a
---
 s/android-4.19/android-base-conditional.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/s/android-4.19/android-base-conditional.xml b/s/android-4.19/android-base-conditional.xml
index 8aab2a0..24e2b47 100644
--- a/s/android-4.19/android-base-conditional.xml
+++ b/s/android-4.19/android-base-conditional.xml
@@ -1,4 +1,4 @@
-<kernel minlts="4.19.191" />
+<kernel minlts="4.19.127" />
 
 <!-- KEEP ALPHABETICALLY SORTED -->
 <!-- ARM base requirements -->
-- 
2.53.0

